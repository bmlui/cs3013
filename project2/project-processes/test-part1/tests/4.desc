evaluate cd to parent directory 
