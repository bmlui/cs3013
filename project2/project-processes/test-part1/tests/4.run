PATH="/bad" ./wshell < tests/4.in
