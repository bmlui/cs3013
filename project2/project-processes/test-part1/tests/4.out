test-part1$ cd tests
tests$ cd ..
test-part1$ exit
